source/configuration.js

```diff
@@ -0,0 +1,44 @@
+
+if(Meteor.isServer) {
+
+  var getenv = Npm.require('getenv');
+  // Wrapper
+  Space.getenv = getenv;
+
+  Space.configuration = Space.getenv.multi({
+    sysLog: {
+      enabled: ['SPACE_SYSLOG_ENABLED', false, 'bool']
+    },
+    appLog: {
+      enabled: ['SPACE_APPLOG_ENABLED', false, 'bool']
+    }
+  });
+
+  // Pass down the
+  Meteor.settings = {
+    "public": {
+      sysLog: {
+        enabled: Space.configuration.sysLog.enabled
+      },
+      appLog: {
+        enabled: Space.configuration.appLog.enabled
+      }
+    }
+  };
+
+  __meteor_runtime_config__.PUBLIC_SETTINGS = Meteor.settings["public"];
```

Comment:
Ah forget it – of course it needs to be more low-level, so that we can log low-level stuff! Or do you think it work otherwise too?